_std/defines/item.dm

```diff
@@ -93,6 +93,8 @@
 #define ITEM_RARITY_LEGENDARY 6
 #define ITEM_RARITY_MYTHIC 7
 
+// item comp defs
+#define GOOD_4_ME_2_EAT 1
```

Comment:
better naming pls

Reply:
Done, now actually named after the thing its supposed to do.